scripts/exregional_make_ics.sh

```diff
@@ -592,6 +592,16 @@ mv_vrfy gfs.bndy.nc ${INPUT_DATA}/${NET}.${cycle}${dot_ensmem}.gfs_bndy.tile${TI
 #
 #-----------------------------------------------------------------------
 #
+# Clean up the external model data files in NCO mode
+#
+#-----------------------------------------------------------------------
+#
+if [ "${RUN_ENVIR}" = "nco" ]; then
+  rm_vrfy -rf ${extrn_mdl_staging_dir}
+fi
+#
```

Comment:
@chan-hoo I am having trouble getting community_ensemble_008mems running properly in NCO mode. Here are issues I am observing
- The above piece of code need to go to the bottom `make_ics.sh` at the very least just like in `make_lbcs.sh` because if `make_ics` fail (which is what is happening), a re-run will not fix it
- Removing the `extrn_mdl_staging_dir` is currently prone to race condition from different ensemble members, so if mem1 deletes it and mem2 hasn't even started running `make_ics`, it will fail
- Can you make it work without `for_ics/lbcs` appended to the staging directory and also keep the input files in COMIN just in case a re-run is needed?